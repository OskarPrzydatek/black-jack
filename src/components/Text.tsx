interface TextProps {
  children: React.ReactNode;
  dataTestID?: string;
}

export default function Text({ children, dataTestID }: TextProps) {
  return <p data-testid={dataTestID}>{children}</p>;
}
