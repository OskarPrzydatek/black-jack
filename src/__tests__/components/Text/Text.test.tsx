import renderer from 'react-test-renderer';

import Text from '../../../components/Text';

describe('Text', () => {
  test('component snapshot', () => {
    const tree = renderer.create(<Text>Lorem Ipsum</Text>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

export {};
