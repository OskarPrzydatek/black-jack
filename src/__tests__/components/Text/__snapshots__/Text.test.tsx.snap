// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text component snapshot 1`] = `
<p>
  Lorem Ipsum
</p>
`;
